python/protobuf_rpc/server.py

```diff
@@ -7,7 +7,8 @@
 
 
 class GServer(ProtoBufRPCServer):
-    def __init__(self, host, port, service, poolsize=128):
+    def __init__(self, host, port, service, logging_format, poolsize=128):
+        super(GServer, self).__init__(logging_format)
```

Comment:
Looking at this again, this is a little awkward because the GServer constructor wasn't really designed to pass args up to ProtoBufRPCServer. Alternative idea is to make the logging format a class-level variable, and if you want to use something other than the default, subclass it. In our case, we're just going to use the default, so that's easy.